src/components/Messaging/Messaging.js

```diff
@@ -123,14 +123,14 @@ function Messaging(props) {
                 let avatar = "";
                 let name = "";
                 let uid = "";
-                conversation.participantAvatars.map(participantAvatar => {
+                conversation.participantAvatars.forEach(participantAvatar => {
```

Comment:
This is a better method for just setting variables.

Reply:
I agree